.pre-commit-hooks.yaml

```diff
@@ -1,12 +0,0 @@
-- args:
```

Comment:
Did dvclive provide pre-commit hooks?

Reply:
Not sure what you mean here. I think this can be safely replaced with `.pre-commit-config.yaml`